packages/base/room.gts

```diff
@@ -576,26 +576,30 @@ export class RoomField extends FieldDef {
             ...cardArgs,
             attachedCardIds,
           });
-        } else if (event.content.msgtype === 'org.boxel.command') {
+        } else if (
+          event.content.msgtype === 'org.boxel.command' &&
+          event.content.data.toolCall
```

Comment:
So this will either need, a technique to deserialize/serialize commands we care about from the matrix messages or delete room message history for staging and production. Former, I wud want to handle in another PR. Latter, I can work on that once this is merged tho maybe people still want to understand staging performance issues and matrix messages are a big part of that.

Reply:
good Q - in general I'm good with in the host app skipping over messages of the old format.